product_manager.py

```diff
@@ -8,19 +10,22 @@ def __init__(self, item_manager, tag_manager, image_manager):
         self.tag_manager = tag_manager
         self.image_manager = image_manager
 
-    def add_product(self, tags, vector_file, png_file):
+    def add_product(self, tags, vector_file, png_file=None):
+        png_file = vector_file if png_file == None else png_file
```

Comment:
can be written
`png_file = vector_file or png_file`
the first truthy value will  be returned

just as with
`a = b and c`
The first falsy value will be returned.

Reply:
updated